apps/modernization-ui/src/apps/search/patient/result/patientSearchResult.spec.tsx

```diff
@@ -89,4 +99,12 @@ describe('patientSearchResult functions', () => {
         expect(getByText('Alias')).toBeInTheDocument();
         expect(getByText('Johnny TestnullTest')).toBeInTheDocument();
     });
+
+    it('should render identifications correctly', () => {
```

Comment:
suggestion (non-blocker):  can you specify what "correctly" is?  This is testing that an identification is displayed titled by it's type

Reply:
Fixed